modules/social_features/social_event/modules/social_event_managers/src/Controller/EventManagersController.php

```diff
@@ -80,9 +81,9 @@ public function access(AccountInterface $account, RouteMatch $route_match) {
       }
     }
 
-    // If we minimize the amount of tabs we can allow LU that can see this
+    // If we minimize the amount of tabs we can allow Verified that can see this
     // event to see the tab as well.
-    if ($node->access('view', $account) && !$account->isAnonymous()) {
+    if ($node->access('view', $account) && !$account->isAnonymous() && !(count($account->getRoles()) == 1 && $account->getRoles()[0] === RoleInterface::AUTHENTICATED_ID)) {
```

Comment:
```suggestion
    if ($node->access('view', $account) && !$account->isAnonymous() && !(count($account->getRoles()) === 1 && $account->getRoles()[0] === RoleInterface::AUTHENTICATED_ID)) {
```

Reply:
Can't we use the `UserHelper->isVerified` function here and negate that?

Reply:
You're right. I changed a little bit method, which checks if a user is a Verified user (`UserHelper->isVerified`), and used that method there.